cronex/__init__.py

```diff
@@ -153,13 +153,18 @@ def compute_numtab(self):
         elif self.string_tab[4] == "*" and self.string_tab[2] != "*":
             self.numerical_tab[4] = set()
 
-    def check_trigger(self, date_tuple, utc_offset=0):
+    def check_trigger(self, date_tuple, utc_offset=0, do_wild=True, do_non_wild=True):
```

Comment:
I generally don't use "do" in my code. This is a function argument, so it's a given that it probably does something.